src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: Snowflake integration
+tags:
+    - Snowflake integration
+    - New Relic integrations
+metaDescription: Install our Snowflake dashboards and see your Snowflake data in New Relic.
+---
+
+import infrastructureSnowflakeDashboard from 'images/infrastructure_screenshot-full_snowflake-dashboard.png'
```

Comment:
are we missing a screenshot? I see two images in this branch but only one import.